src/atopile/cli/inspect.py

```diff
@@ -32,68 +33,59 @@
 
 log = logging.getLogger(__name__)
 log.setLevel(logging.INFO)
-
 class DisplayEntry:
-    def __init__(self, connectables: list[AddrStr]):
-        self.connectables: list[AddrStr] = connectables
-        self.intermediate_connectable: list[AddrStr] = []
-        self.consumers: list[AddrStr] = []
-
-
-class InspectConnectable:
-    def __init__(self, addr: AddrStr, type: str):
-        self.addr = addr
-        self.type = type
-        self.associated_connectable = []
-        self.parent_interface = None
-        self.consumer = []
-
-
-def collect_connected_dfs(connections: list[Link], root_addr, visited=None) -> list[AddrStr]:
-    """
-    from a list of links and a root node, this function returns all the nodes connected to the root node
-    """
-    if visited is None:
-        visited = []
-
-    for connection in connections:
-        if connection.source.addr == root_addr:
-            if connection.target.addr not in visited:
-                visited.append(connection.target.addr)  # Mark this target as visited
-                # Recursively visit the targets connected to this target
-                collect_connected_dfs(connections, connection.target.addr, visited)
-        elif connection.target.addr == root_addr:
-            if connection.source.addr not in visited:
-                visited.append(connection.source.addr)  # Mark this target as visited
-                # Recursively visit the targets connected to this target
-                collect_connected_dfs(connections, connection.source.addr, visited)
-    return visited
-
-def find_nets(graph) -> list[list[AddrStr]]:
     """
-    For a dict of connections:
-    {
-    A: [B, C],
-    B: [A, C],
-    C: [A, B],
-    D: [E],
-    E: [D]
-    }
-    This function returns the nets:
-    [[A, B, C], [D, E]]
+    This class represents the nets that are below the inspected module,
+    the equivalent net that is below the context module and
+    the individual connections that are made to the inspect net and the context net.
     """
-    visited = set()  # Keep track of visited nodes to avoid revisiting them.
-    connected_components = []  # Store the connected components.
+    def __init__(self, net: list[list[AddrStr]]):
+        self.inspect_net: list[AddrStr] = net
+        self.inspect_consumer: list[AddrStr] = []
+        self.context_net: list[AddrStr] = []
+        self.context_consumer: list[AddrStr] = []
+
+def find_nets_new(links: list[Link]) -> list[list[AddrStr]]:
```

Comment:
nit; it's generally better to leave things like "new" out of names like this. Has little meaning and you don't have "find_nets" in this scope already. In this case, I'd probably also make it private (prefix it with `_`)